packages/remediations/src/newSteps/issueResolution.scss

```diff
@@ -21,6 +21,10 @@
             }
         }
     }
+    p.ins-c-playbook-description {
+        margin-bottom: var(--pf-global--spacer--sm);
```

Comment:
You can use PF4 [utilities](https://www.patternfly.org/v4/utilities/text) `className="pf-u-mb-sm pf-u-font-size-sm"` instead of custom css.